pkg/tests/ossm/operator/clusterwide_mode_test.go

```diff
@@ -339,6 +411,22 @@ spec:
 	})
 }
 
+func assertResponseCode(t test.TestHelper, from string, to string, code string) {
+	curlFromTo(t, from, to,
+		assert.OutputContains(code,
+			fmt.Sprintf("%s -> %s, response %s", from, to, code),
+			fmt.Sprintf("%s -> %s, response %s,  but shouldn't", from, to, code)))
+}
+
+func curlFromTo(t test.TestHelper, from string, to string, checks ...common.CheckFunc) {
+	output := oc.Exec(t,
+		pod.MatchingSelector("app=sleep", from),
+		"sleep",
+		fmt.Sprintf(`curl http://httpbin.%s:8000/ip --connect-timeout 5 -s -o /dev/null -w "sleep.%s to httpbin.%s: %%{http_code}" || echo "CURL_FAILED"`, to, from, to),
+		checks...)
+	t.Log(output)
+}
+
```

Comment:
This is similar to this already used in another test case: https://github.com/maistra/maistra-test-tool/blob/bcb0b8c155790cd1dfa0b837d93b6be36edcb3cf/pkg/tests/tasks/security/authentication/mtls_migration_test.go#L103-L123

Maybe worth to extract this function to the proper package and used in all the test cases needed as a global function available in the func

Reply:
It is a good idea, I will make a separate Jira for that small refactoring because there are a few more tests using same command/curl inside sleep. For now, I have replaced connection test with mtls header test because connections is longer